unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideoTexture.cs

```diff
@@ -45,7 +45,17 @@ public class DCLVideoTexture : DCLTexture
 
         internal Dictionary<string, MaterialInfo> attachedMaterials = new Dictionary<string, MaterialInfo>();
 
-        public DCLVideoTexture() { model = new Model(); }
+        public DCLVideoTexture()
+        {
+            model = new Model();
+
+            DataStore.i.virtualAudioMixer.sceneSFXVolume.OnChange += OnVirtualAudioMixerChangedValue;
+        }
+
+        ~DCLVideoTexture()
+        {
+            DataStore.i.virtualAudioMixer.sceneSFXVolume.OnChange -= OnVirtualAudioMixerChangedValue;
+        }
```

Comment:
Please don't use finalizers, they are not recommended. Use the `Dispose()` pattern instead.